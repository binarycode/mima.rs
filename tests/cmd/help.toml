args = 'help'
status.code = 0
