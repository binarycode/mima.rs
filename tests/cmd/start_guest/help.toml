args = 'help start-guest'
status.code = 0
