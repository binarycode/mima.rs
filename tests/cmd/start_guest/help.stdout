mima-start-guest [VERSION]
Start guest

USAGE:
    mima start-guest [OPTIONS] <GUEST_ID>

ARGS:
    <GUEST_ID>    Guest ID

OPTIONS:
        --boot-from-cdrom         Boot from CD-ROM
        --boot-from-network       Boot from network
        --cdrom <CDROM_PATH>      Insert CD-ROM image from specified path
        --floppy <FLOPPY_PATH>    Insert floppy image from specified path
    -h, --help                    Print help information
